controller.py

```diff
@@ -283,10 +283,14 @@ def register(self, do_get=HTTP.get_with_timeout):
 
         integration_contact_uri = flask.request.form.get("contact")
         integration_contact_email = self._required_email_address(
-            integration_contact_uri, "No valid integration contact address"
+            integration_contact_uri,
+            "Invalid or missing configuration contact email address"
         )
         if isinstance(integration_contact_email, ProblemDetail):
             return integration_contact_email
+        hyperlinks_to_create = [
+            (Hyperlink.INTEGRATION_CONTACT_REL, [integration_contact_email])
+        ]
```

Comment:
We build up a list of Hyperlink objects we need to create, and then create them by calling set_hyperlink once a Library object exists.